sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterLoggingExtensions.cs

```diff
@@ -27,6 +27,10 @@ public static OpenTelemetryLoggerOptions AddAzureMonitorLogExporter(this OpenTel
                 throw new ArgumentNullException(nameof(loggerOptions));
             }
 
+            // Ideally user should set this to true
```

Comment:
maybe - ideally SDK should make it true by default, or the user should..

Reply:
will open an issue on SDK side to track this.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/4213